api/xverse.ts

```diff
@@ -92,6 +93,34 @@ export async function getCoinsInfo(
     });
 }
 
+/**
+ * get BRC-20 supported tokens with the fiat rate
+ * @param network
+ * @param tickers provided to get the fiat rate along with supported tokens
+ * @param fiatCurrency
+ */
+export async function getBrc20Tokens(
+  network: NetworkType,
+  tickers: string[],
+  fiatCurrency: string,
+): Promise<Brc20TokensResponse | null> {
+  const url = `${XVERSE_API_BASE_URL(network)}/v1/brc20/tokens`;
+
+  const params = {
+    currency: fiatCurrency,
+    tickers: tickers,
```

Comment:
note: need to double check that this gets serialised into the query params properly

Reply:
oh yeah this format is not supported:

http://localhost:3001/v1/brc20/tokens?currency=USD&tickers[]=TTIN&tickers[]=SA